index.html

```diff
@@ -154,6 +154,258 @@ <h2 class="sub-title">Software Developer</h2>
+        <g filter="url(#filter0_f_48_2353)">
+          <ellipse rx="35.1406" ry="130.801" transform="matrix(-0.83914 -0.543915 -0.543915 0.83914 127.633 161.874)" fill="#EDECFF" />
+        </g>
+        <defs>
+          <filter id="filter0_f_48_2353" x="0.599121" y="0.443405" width="254.068" height="322.861" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+            <feFlood flood-opacity="0" result="BackgroundImageFix" />
+            <feBlend mode="normal" in="SourceGraphic" in2="BackgroundImageFix" result="shape" />
+            <feGaussianBlur stdDeviation="25" result="effect1_foregroundBlur_48_2353" />
+          </filter>
+        </defs>
+      </svg>
+    </div>
+    <div class="card-bg-img-3">
+      <svg width="265" height="255" viewBox="0 0 265 255" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <g filter="url(#filter0_f_48_2352)">
+          <ellipse cx="61" cy="127.5" rx="154" ry="77.5" fill="#EDECFF" fill-opacity="0.3" />
+        </g>
+        <defs>
+          <filter id="filter0_f_48_2352" x="-143" y="0" width="408" height="255" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+            <feFlood flood-opacity="0" result="BackgroundImageFix" />
+            <feBlend mode="normal" in="SourceGraphic" in2="BackgroundImageFix" result="shape" />
+            <feGaussianBlur stdDeviation="25" result="effect1_foregroundBlur_48_2352" />
+          </filter>
+        </defs>
+      </svg>
+    </div>
+    <div class="card-bg-img-4">
+      <svg width="181" height="239" viewBox="0 0 181 239" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <g filter="url(#filter0_f_48_2351)">
+          <ellipse rx="50.6467" ry="188.518" transform="matrix(-0.258819 0.965926 0.965926 0.258819 233.203 119.713)" fill="#EDECFF" />
+        </g>
+        <defs>
+          <filter id="filter0_f_48_2351" x="0.629395" y="0.619385" width="465.147" height="238.187" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+            <feFlood flood-opacity="0" result="BackgroundImageFix" />
+            <feBlend mode="normal" in="SourceGraphic" in2="BackgroundImageFix" result="shape" />
+            <feGaussianBlur stdDeviation="25" result="effect1_foregroundBlur_48_2351" />
+          </filter>
+        </defs>
+      </svg>
+    </div>
+
+    <div class="projects-heading">
+      <h2>MY RECENT WORK</h2>
+    </div>
+    <div class="devider">
+    </div>
+    <div class="projects-container">
+      <div class="project-card">
+        <div class="card-img">
+          <img src="project-img.png" alt="">
+        </div>
+        <div class="card-title">
+          <h4>Multi-Post Stories</h4>
+        </div>
+        <div class="card-text">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+          has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </div>
```

Comment:
- [ ] Please consider using a `p` tag here instead of the `div` tag, this is because the content is actually a paragraph and should be put in a paragraph tag `p`. This is to have a semantic HTML workflow. learn more about the `p` tag [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p)